src/avram/tasks/db/create.cr

```diff
@@ -19,8 +20,22 @@ class Db::Create < LuckyCli::Task
   end
 
   def call
-    Avram::Migrator.run do
-      Avram::Migrator::Runner.create_db(@quiet)
+    AM::Runner.run("psql -q -l #{connection_url}", output: IO::Memory.new)
```

Comment:
So this just tries to connect but doesn't really do anything right? If so what do you think about extracting a private def called `attempt_to_connect`

Reply:
Yeah, it just hops in to the psql. If it's able to, then we have a connection to the engine which means we can run db.create. I like that. makes it clear.